server/app/src/main/kotlin/server/route/handler/GetMeRouteHandler.kt

```diff
@@ -24,7 +24,7 @@ class GetMeRouteHandler(
 
     @Serializable
     private data class GetMeResponseBody(
-        val displayName: String,
+        @Serializable(with = UUIDSerializer::class) val uuid: UUID,
```

Comment:
It might be worth being specific with the naming (like `userUuid`), to make it clearer for people coming across our models for the first time?